pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -354,9 +355,10 @@ type containerToKillInfo struct {
 	message string
 }
 
-// podActions keeps information what to do for a pod.
+// podActions keeps information what to do for a pod based on its spec.
+// Note that podActions will not affect Debug Containers since they are not part of the spec.
 type podActions struct {
-	// Stop all running (regular and init) containers and the sandbox for the pod.
+	// Stop all running (regular, init and debug) containers and the sandbox for the pod.
```

Comment:
This seems to conflict with the above comment stating that the actions don't affect debug containers.